src/app/main/component/ubs/components/ubs-main-page/ubs-main-page.component.scss

```diff
@@ -0,0 +1,323 @@
+  background-image: url(/assets/img/ubs/first_rectangle.png);
+  background-size: 184px 282px;
+}
+
+.img_2 {
+  width: 79px;
+  height: 230px;
+  position: absolute;
+  top: 20px;
+  left: 55px;
+}
+
+.header-price {
+  position: absolute;
+  text-align: center;
+  z-index: 2;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 28px;
+  align-self: start;
+  margin: 30% 0 10px;
+}
+
+.content-price {
+  position: absolute;
+  text-align: center;
+  z-index: 2;
+  font-size: 16px;
+  line-height: 24px;
+  align-self: start;
+  margin: 50% 25px 10px;
+  padding-top: 10px;
+}
+
+.boxCard {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: 1440px;
+  justify-content: center;
+}
+
+.box {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+}
+
+#step-order {
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+  flex: 1 0 25%;
+  width: 133px;
+  height: 203px;
+  margin: 50px;
+}
+
+.circle {
+  position: relative;
+  width: 30px;
+  height: 30px;
+  line-height: 30px;
+  background-color: var(--ubs-tertiary-dark-grey);
+  color: var(--ubs-primary-white);
+  margin: 10px;
+  border-radius: 50%;
+  text-align: center;
+  font-size: 20px;
+}
+
+hr {
+  border: none;
+  background-color: var(--ubs-tertiary-light-grey);
+  height: 2px;
+  width: 72px;
+  position: absolute;
+  left: 55px;
+  top: 5px;
+}
+
+.header-step {
+  position: absolute;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 28px;
+  align-self: start;
+  margin: 30% 10px 10px;
+}
+
+.content-step {
+  position: absolute;
+  font-size: 14px;
+  line-height: 28px;
+  align-self: start;
+  margin: 100px 10px 120px 10px;
+  padding-top: 20px;
+  width: 160px;
+}
+
+#how-works {
+  margin: 50px 50px 50px auto;
+}
+
+#header-how-works {
+  font-size: 36px;
+  font-weight: bold;
+  line-height: 44px;
+  margin-bottom: 25px;
+}
+
+.time-how {
+  font-size: 18px;
+  font-weight: bold;
+  margin: 1px 25px 1px auto;
+}
+
+#how-works-content {
+  margin: 15px 10px 15px 0;
+}
+
+@media screen and (min-width: 770px) {
+  #step-order {
+    height: 250px;
+    margin: 50px auto;
+  }
+
+  #price-frame {
+    width: 250px;
+    height: 228px;
+    background-size: 250px 228px;
+  }
+
+  .img_2 {
+    height: 189px;
+    width: 108px;
+    top: 15px;
+    left: 75px;
+  }
+
+  .header-price {
+    margin-top: 50px;
+  }
+
+  .content-price {
+    margin: 80px 40px 10px;
+  }
+}
+
+@media screen and (min-width: 1024px) {
+  .ubsHomepage {
+    margin: auto 12%;
+  }
+
+  #price-frame {
+    width: 310px;
+    height: 178px;
+    background-size: 310px 187px;
+  }
+
+  .img_2 {
+    height: 150px;
+    width: 163px;
+    top: 15px;
+    left: 75px;
+  }
+
+  .header-price {
+    margin-top: 40px;
+  }
+
+  .content-price {
+    margin: 70px 25px 10px;
+  }
+
+  .content-step {
+    margin-top: 110px;
+  }
+}
+
+@media screen and (max-width: 425px) {
+  #price-frame {
+    width: 310px;
+    height: 178px;
+    background-size: 310px 187px;
```

Comment:
These lines of code are repeating a lot of times in this file.
I suggest making @mixin